pkg/network/usm/ebpf_gotls.go

```diff
@@ -40,6 +41,10 @@ const (
 	goTLSReadArgsMap          = "go_tls_read_args"
 	goTLSWriteArgsMap         = "go_tls_write_args"
 	connectionTupleByGoTLSMap = "conn_tup_by_go_tls_conn"
+
+	// The interval of the periodic scan for terminated processes. Increasing the interval, might cause larger spikes in cpu
+	// and lowering it might cause constant cpu usage.
+	scanTerminatedProcessesInterval = 30 * time.Second
```

Comment:
that const is not used in that file, should we move it to another place?

Reply:
I'm not sure if I follow you. the constant _is_ used in this file:
https://github.com/DataDog/datadog-agent/blob/01e28493c3981bdcea3bb708baaa568274907989/pkg/network/usm/ebpf_gotls.go#L275